src/bin/frontendlauncher.py

```diff
@@ -234,16 +234,25 @@ def ParseVersion(ver):
     ver = ""
 
     # The version that we use for plugins. Since we have a development
-    # version, let's try to get the version from the VERSION file.
+    # version, let's try to get the VisIt version from various files it might
+    # be in. The file cases we try here have grown as we've moved where we
+    # store VisIt version information.
     visitpluginver = ""
     VERSIONFILE = visitdir + "VERSION"
     VISIT_CONFIG_H = visitdir + "include" + os.path.sep + "visit-config.h"
+    VISIT_VERSION_H = visitdir + "include" + os.path.sep + "visit-version.h"
     if os.path.exists(VERSIONFILE):
         visitpluginver = open(VERSIONFILE).readlines()[0][:-1]
-    elif os.path.exists(VISIT_CONFIG_H):
+    if not visitpluginver and os.path.exists(VISIT_CONFIG_H):
```

Comment:
Just to verify: `if not visitpluginver` tests whether the string is empty?

Reply:
Yes. Empty string in Python is equiv to `False`